bcda/api.go

```diff
@@ -149,67 +148,43 @@ func bulkRequest(t string, w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	var acoBeneficiaries []models.ACOBeneficiary
-	if err = db.Find(&acoBeneficiaries, "aco_id = ?", acoID).Error; err != nil {
-		log.Errorf("Error retrieving ACO-beneficiary with ACO ID %s: %s", acoID, err.Error())
-		oo := responseutils.CreateOpOutcome(responseutils.Error, responseutils.Exception, "", responseutils.DbErr)
-		responseutils.WriteError(oo, w, http.StatusInternalServerError)
-		return
-	}
-
-	beneficiaryIDs := []string{}
-	for _, acoBeneficiary := range acoBeneficiaries {
-		var beneficiary models.Beneficiary
-		if err = db.First(&beneficiary, acoBeneficiary.BeneficiaryID).Error; err != nil {
-			log.Errorf("Error retrieving beneficiary with ID %d: %s", acoBeneficiary.BeneficiaryID, err.Error())
-			oo := responseutils.CreateOpOutcome(responseutils.Error, responseutils.Exception, "", responseutils.DbErr)
-			responseutils.WriteError(oo, w, http.StatusInternalServerError)
-			return
-		}
-		beneficiaryIDs = append(beneficiaryIDs, beneficiary.BlueButtonID)
-	}
-
 	// TODO: this checks for ?encrypt=false appended to the bulk data request URL
 	// By default, our encryption process is enabled but for now we are giving users the ability to turn
 	// it off
 	// Eventually, we will remove the ability for users to turn it off and it will remain on always
-	var encrypt = true
+	var encrypt bool = true
```

Comment:
`bool` is unnecessary because it's implied by the value. I think it used to be there, but VSCode complained so I removed it.

Reply:
will remove in incoming commit.